integration-tests/gatsby-source-wordpress/test-fns/data-resolution.js

```diff
@@ -45,7 +45,7 @@ describe(`data resolution`, () => {
     expect(data[`allWpMenu`].totalCount).toBe(3)
     expect(data[`allWpMenuItem`].totalCount).toBe(4)
     expect(data[`allWpPostFormat`].totalCount).toBe(0)
-    expect(data[`allWpContentType`].totalCount).toBe(9)
+    expect(data[`allWpContentType`].totalCount).toBe(7)
```

Comment:
What's the source of this change?

Reply:
Swapping wp-graphql-gutenberg extension to wp-graphql-content-blocks, previous plugin created actual wp content types, while new one does not